seqr/utils/search/utils.py

```diff
@@ -255,8 +251,11 @@ def _query_variants(search_model, user, previous_search_results, sort=None, num_
     dataset_type, secondary_dataset_type, lookup_dataset_type = _search_dataset_type(parsed_search)
     parsed_search.update({'dataset_type': dataset_type, 'secondary_dataset_type': secondary_dataset_type})
     search_dataset_type = None
-    if dataset_type and dataset_type != ALL_DATA_TYPES and (secondary_dataset_type is None or secondary_dataset_type == dataset_type):
-        search_dataset_type = lookup_dataset_type or dataset_type
+    if dataset_type and dataset_type != ALL_DATA_TYPES:
+        if secondary_dataset_type is None or secondary_dataset_type == dataset_type:
+            search_dataset_type = lookup_dataset_type or dataset_type
+        elif dataset_type == Sample.DATASET_TYPE_SV_CALLS:
+            search_dataset_type = DATASET_TYPE_NO_MITO
```

Comment:
what's the purpose of this `DATASET_TYPE_NO_MITO` `search_dataset_type`?

Reply:
also, there are no mitochondrial SVs so we do not look for compound hets that are an SV/MITO pair. So if its a recessive search where the primary data type is SV, then no mito variants can be the primary variant, and no mito variants can be the secondary variant because they will not share a gene by definition, so this is handling that case and saying we should not include the MITO data here either